### PR TITLE
Load JS on every page to be able to parse URLs

### PR DESCRIPTION
**Pull request by torotil**

### CODE REVIEW

File: webform_prefill.js [hunk 1]
```diff
@@ -7,17 +7,13 @@
 
 Drupal.behaviors.webform_prefill = {
   attach: function (context, settings) {
-
-    if (!$.isFunction($.fn.formPrefill)) {
-      return;
-    }
-
-    var settings = Drupal.settings.webform_prefill || {
+    var settings = settings.webform_prefill || {
```
Comment (katzmo) [inline webform_prefill.js]: Why can you be sure that `$.fn.formPrefill` exists but not `settings.webform_prefill`? Both of them should have been added in `webform_prefill_page_build()`?

File: webform_prefill.js [hunk 2]
```diff
@@ -7,17 +7,13 @@
...
       map: {},
       storage: ['sessionStorage'],
       cookieDomain: ''
     };
 
+    // This also triggers reading values from the URL even if there is no form.
```
Comment (katzmo) [inline webform_prefill.js:12]: How does that work when `$('form.webform-client-form', context)` is empty?

Comment (torotil) [reply webform_prefill.js:12]: `$.fn.formPrefill` is still called on the empty result. It does most of its work *before* attempting to loop over the elements.

File: webform_prefill.js (hunk 1 above)
Comment (torotil) [reply webform_prefill.js]: Hm, yes perhaps we can remove that.

File: webform_prefill.js (hunk 2 above)
Comment (katzmo) [reply webform_prefill.js:12]: I'm pretty sure this is against some best practice :rofl:

From the [jQuery learning center](https://learn.jquery.com/plugins/):
"The idea of a plugin is to do something with a collection of elements."

**Review by katzmo — APPROVED** [review]

**Pull request merged**